spyder/plugins/editor/extensions/docstring.py

```diff
@@ -67,6 +102,39 @@ def is_beginning_triple_quotes(text):
 
         return False
 
+    def is_end_of_function_definition(self, text, lineNumber):
+        """Return True if text is the end of the function definition."""
+        textWithoutWhitespace = "".join(text.split())

```

Comment:
Please change the name of this variable to be

    text_without_whitespace

That's because we only use camel case variables to name classes.